test/Conversion/TorchToLinalg/view.mlir

```diff
@@ -0,0 +1,76 @@
+// CHECK:    return %[[BUILTIN_TENSOR_CAST]] : !torch.vtensor<[2,3],f32>
+
+func.func @torch.aten.view$twotothree(%ARG: !torch.vtensor<[3,2],f32>) -> !torch.vtensor<[2,3],f32> {
+    %0 = torch.constant.int 3
+    %1 = torch.constant.int 2
+    %LIST = torch.prim.ListConstruct %1, %0 : (!torch.int, !torch.int) -> !torch.list<int>
+    %VIEW = torch.aten.view %ARG, %LIST : !torch.vtensor<[3,2],f32>, !torch.list<int> -> !torch.vtensor<[2,3],f32>
+    return %VIEW : !torch.vtensor<[2,3],f32>
+  }
+
+// CHECK-LABEL: func.func @torch.aten.view$dynamictest(
+// CHECK-SAME:      %[[ARG:.*]]: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[?,?],f32> {
+// CHECK:        %[[BUILTIN_TENSOR:.*]] = torch_c.to_builtin_tensor %[[ARG]] : !torch.vtensor<[?,?],f32> -> tensor<?x?xf32>
+// CHECK:        %[[CASTED:.*]] = tensor.cast %[[BUILTIN_TENSOR]] : tensor<?x?xf32> to tensor<?x?xf32>
+// CHECK:        %[[BUILTIN_TENSOR_CAST:.*]] = torch_c.from_builtin_tensor %[[CASTED]] : tensor<?x?xf32> -> !torch.vtensor<[?,?],f32>
+// CHECK:        return %[[BUILTIN_TENSOR_CAST]] : !torch.vtensor<[?,?],f32>
+
+func.func @torch.aten.view$dynamictest(%ARG: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[?,?],f32> {
+    %0 = torch.constant.int 1
+    %1 = torch.constant.int 0
+    %LISTVALUE1 = torch.aten.size.int %ARG, %1 : !torch.vtensor<[?,?],f32>, !torch.int -> !torch.int
+    %LISTVALUE2 = torch.aten.size.int %ARG, %0 : !torch.vtensor<[?,?],f32>, !torch.int -> !torch.int
+    %LIST = torch.prim.ListConstruct %LISTVALUE1, %LISTVALUE2 : (!torch.int, !torch.int) -> !torch.list<int>
+    %VIEW = torch.aten.view %ARG, %LIST : !torch.vtensor<[?,?],f32>, !torch.list<int> -> !torch.vtensor<[?,?],f32>
+    return %VIEW : !torch.vtensor<[?,?],f32>
+  }
+
+// CHECK-LABEL: func.func @torch.aten.view$dynamicVal(
+// CHECK-SAME:     %[[ARG:.*]]: !torch.vtensor<[1,?,128],f32>) -> !torch.vtensor<[16,1,128],f32> {
+// CHECK:     %[[BUILTIN_TENSOR:.*]] = torch_c.to_builtin_tensor %[[ARG]] : !torch.vtensor<[1,?,128],f32> -> tensor<1x?x128xf32>
+// CHECK:     %[[CASTED:.*]] = tensor.cast %[[BUILTIN_TENSOR]] : tensor<1x?x128xf32> to tensor<1x16x128xf32>
+// CHECK:     %[[COLLAPSED:.*]] = tensor.collapse_shape %[[CASTED]] {{\[\[}}0, 1], [2]] : tensor<1x16x128xf32> into tensor<16x128xf32>
+// CHECK:     %[[EXPANDED:.*]] = tensor.expand_shape %[[COLLAPSED]] {{\[\[}}0], [1, 2]] : tensor<16x128xf32> into tensor<16x1x128xf32>
+// CHECK:     %[[EXPAND_CAST:.*]] = tensor.cast %[[EXPANDED]] : tensor<16x1x128xf32> to tensor<16x1x128xf32>
+// CHECK:     %[[BUILTIN_TENSOR_CAST:.*]] = torch_c.from_builtin_tensor %[[EXPAND_CAST]] : tensor<16x1x128xf32> -> !torch.vtensor<[16,1,128],f32>
+// CHECK:     return %[[BUILTIN_TENSOR_CAST]] : !torch.vtensor<[16,1,128],f32>
+
+func.func @torch.aten.view$dynamicVal(%ARG: !torch.vtensor<[1,?,128],f32>) -> !torch.vtensor<[16,1,128],f32> {
+    %I128 = torch.constant.int 128
```

Comment:
can you please follow the standard MLIR naming conventions here? You should be able to just run this file through torch-mlir-opt to pretty print it correctly.

Reply:
This looks unresolved. I still see variable names like `%LIST` and stuff.

Reply:
oh, I misunderstood what you wanted me to do, sorry about that.